src/qt/veil/forms/miningwidget.ui

```diff
@@ -308,6 +250,47 @@ QSpinBox::down-button { width: 24px; height: 24px;}</string>
             </layout>
            </widget>
           </item>
+          <item>
+           <widget class="QFrame" name="frame_8">
+            <property name="minimumSize">
+             <size>
+              <width>0</width>
+              <height>40</height>
+             </size>
+            </property>
+            <property name="frameShape">
+             <enum>QFrame::StyledPanel</enum>
+            </property>
+            <property name="frameShadow">
+             <enum>QFrame::Raised</enum>
+            </property>
+            <layout class="QHBoxLayout" name="horizontalLayout_8">
+             <item>
+              <widget class="QLabel" name="lblExceedThr">
+               <property name="minimumSize">
+                <size>
+                 <width>300</width>
+                 <height>0</height>
+                </size>
+               </property>
+               <property name="styleSheet">
+                <string notr="true">color:red;
+font-size:16px;</string>
+               </property>
+               <property name="text">
+                <string>Exceeds recommeded number of threads</string>
```

Comment:
spelling issue: s/b "recommended".  This is visible to the user.

Reply:
Fixed